Lykke.Service.LykkeService/client/Lykke.Service.LykkeService.Client/LykkeServiceClient.cs

```diff
@@ -4,12 +4,18 @@ namespace Lykke.Service.LykkeService.Client
 {
     public class LykkeServiceClient : ILykkeServiceClient
```

Comment:
What is the use of this class when you use refit? A wrapper for?

Reply:
This is a proxy for ILykkeServiceClient that can:
- expose ILykkeServiceClient either directly
- or serve as Facade (routing client interface calls to mutiple controllers
- or expose controllers interfaces via public properties

Reply:
Not sure if in most cases proxy is needed for clients generated by HttpClientGenerator. More often this class will be used as aggregation root for multiple APIs, I believe.